### PR TITLE
Cymulate WAF v1.0.0

### PR DESCRIPTION
**Pull request by cs-omkar-kumbhar**
#### Descriptions:
PMDB: 30320

#### UTCs:
- [x] Connector installation verified.
- [x] Connector logo verified.
- [x] Actions and Playbook list verified.
- [x] Playbook tags verified.
- [x] All Playbook are in info mode verified.
- [x] All Playbook are in inactive mode verified.

### CODE REVIEW

File: cymulate-web-application-firewall/connector.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+class CymulateWebApplicationFirewall(Connector):
+    def execute(self, config, operation, params, **kwargs):
+        try:
+            operation = operations.get(operation)
+        except Exception as err:
+            logger.exception(err)
+            raise ConnectorError(err)
+        return operation(config, params)
```
Comment (cs-sameer-pathan) [inline cymulate-web-application-firewall/connector.py]: Please move this line in try block

File: cymulate-web-application-firewall/info.json [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+        "type": "password",
+        "editable": true,
+        "visible": true,
+        "required": true
+      },
+      {
+        "title": "Verify SSL",
+        "name": "verify_ssl",
+        "description": "Specifies whether the SSL certificate for the server is to be verified. By default, this option is selected, i.e., set to True.",
+        "tooltip": "Specifies whether the SSL certificate for the server is to be verified. By default, this option is selected, i.e., set to True.",
```
Comment (cs-sameer-pathan) [inline cymulate-web-application-firewall/info.json]: By default, this option is selected, i.e., set to True.", remove from tooltip

File: cymulate-web-application-firewall/info.json [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+      "title": "Get WAF Payload Response",
+      "description": "Retrieves the Web Application Firewall response data based on the payload ID that you have specified.",
+      "category": "investigation",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Payload ID",
+          "name": "id",
+          "type": "text",
+          "description": "Specify the payload ID",
```
Comment (cs-sameer-pathan) [inline cymulate-web-application-firewall/info.json]: Update description and tooltip to a complete sentence, for example
Specify the payload ID to whose Web Application Firewall response data is to be retrieved.

Comment (cs-sameer-pathan) [reply cymulate-web-application-firewall/info.json]: Please do same for other parameters as well

File: cymulate-web-application-firewall/info.json [hunk 4]
```diff
@@ -0,0 +1,405 @@
...
+          "title": "Schedule",
+          "name": "schedule",
+          "type": "datetime",
+          "description": "Specify the datetime of the schedule.",
+          "tooltip": "Specify the datetime of the schedule.",
+          "required": false,
+          "visible": true,
+          "editable": true
+        },
+        {
```
Comment (cs-sameer-pathan) [inline cymulate-web-application-firewall/info.json:194]: Please check if this parameter can be changed to select , if not check if other options could be suggested

**Review by cs-sameer-pathan — CHANGES_REQUESTED** [review]

File: cymulate-web-application-firewall/connector.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
+"""
+Copyright start
+Copyright (C) 2008 - 2024 FortinetInc.
+All rights reserved.
+FORTINET CONFIDENTIAL & FORTINET PROPRIETARY SOURCE CODE
```
Comment (cs-sameer-pathan) [inline cymulate-web-application-firewall/connector.py]: Please check is this copyright correct

Comment (cs-omkar-kumbhar) [reply cymulate-web-application-firewall/connector.py]: yes, it is used in some other connectors as well

File: cymulate-web-application-firewall/info.json (hunk 4 above)
Comment (cs-omkar-kumbhar) [reply cymulate-web-application-firewall/info.json:194]: this is uncertified connector and in the doc also any information is not provided for this parameter

**Pull request merged**